### PR TITLE
Google Analytics and status check page

### PR DESCRIPTION
**Pull request by cmmartti**
Add status page with links to updown.io
Add Google Analytics
Disable source maps in production
Update outage alert

### CODE REVIEW

**Comment by tdmalone** [comment]
Have reviewed + approved to avoid blocking, but will leave the merge for you when you're ready in case you want to wait for @Naramsim too.

File: alerts.json [hunk 1]
```diff
@@ -15,6 +15,6 @@
     {
         "active": true,
         "important": true,
-        "message": "**Outage Alert:** Named endpoints for PokéAPI v2 are currently out of service. We are investigating. [Read more...](https://github.com/PokeAPI/pokeapi/issues/369)"
+        "message": "**Outage Alert:** Named endpoints and `offset`/`limit` pagination for PokéAPI v2 are currently broken. We are investigating. [Read more...](https://github.com/PokeAPI/pokeapi/issues/369)"
```
Comment (Naramsim) [inline alerts.json]: Named are back up :)

**Comment by Naramsim** [comment]
Good to merge

**Comment by cmmartti** [comment]
Do you want to push it to the DO server or should I?

**Comment by Naramsim** [comment]
I can push it

**Pull request merged**